kratos/utilities/coordinate_transformation_utilities.h

```diff
@@ -23,6 +23,7 @@
 // kratos includes
 #include "includes/define.h"
 #include "includes/node.h"
+#include "includes/model_part.h"
```

Comment:
ouch this is a heavy include
AFAIK this file is included in many elements no?

=> Why do you include it here?

Reply:
I have a utilitiy which only uses geomerty as in this utility, then `ModelPart` becomes an incomplete type and compilation fails.

Reply:
ok I checked, this is not included in elements

also the ModelPart is being used in there for a while already, then all good